openmc/deplete/abc.py

```diff
@@ -523,8 +525,10 @@ class Integrator(ABC):
         Power density of the reactor in [W/gHM]. It is multiplied by
         initial heavy metal inventory to get total power if ``power``
         is not speficied.
+    flux : float or iterable of float, optional
+        neutron flux in [neut/s-cm^2] for each interval in :attr:`timesteps`
```

Comment:
Since this is being treated exactly the same as `source_rates`, I'm not sure we should introduce a new argument just for this. We can expand the description of `source_rates` below to say that it can also represent the neutron flux (which sort of _is_ the source rate in a sense) and then also make it clear in the .rst documentation.

Reply:
I agree I think we can just add more spec to `source_rates`. Where in the .rst documentation are you referring to?